pyproject.toml

```diff
@@ -81,6 +82,27 @@ yt-dlp = "yt_dlp:main"
 [project.entry-points.pyinstaller40]
 hook-dirs = "yt_dlp.__pyinstaller:get_hook_dirs"
 
+[tool.hatch.envs.default]
+dependencies = ["yt-dlp[dev]"]
+
+[tool.hatch.envs.default.scripts]
```

Comment:
IMO all the `envs` tables should be moved below `tool.hatch.version` -- keep project metadata and distribution stuff at the top, env/tooling config below